code/modules/mob/living/silicon/silicon.dm

```diff
@@ -340,3 +340,10 @@
 	for(var/obj/machinery/camera/C in A.cameras())
 		cameratext += "[(cameratext == "")? "" : "|"]<A HREF=?src=\ref[src];switchcamera=\ref[C]>[C.c_tag]</A>"
 	src << "[A.alarm_name()]! ([(cameratext)? cameratext : "No Camera"])"
+
+/mob/living/silicon/adjustToxLoss(var/amount)
+	return
+
+/mob/living/silicon/adjustOxyLoss(var/amount)

```

Comment:
AI's veryyyy much use this, this is going to make them invincible to not having power.